motioneye/static/js/main.js

```diff
@@ -5084,6 +5093,20 @@ function addCameraFrameUi(cameraConfig) {
         }
     };
 
+    /* check if action buttons need one or two rows*/
+    if(cameraConfig.actions.length) {
+        var actionButtonWidth = actionButtonDict[cameraConfig.actions[0]].width();
+        var actionsWidth = actionsCount * actionButtonWidth;
+        var cameraWidth = cameraOverlay.find('div.camera-overlay-bottom').width();
+        var flexBasis = 200 / actionsCount;
+        flexBasis += '%';
+        if(cameraWidth <= actionsWidth){
+            actionButtonWrapper.css('flex', '1 0' + flexBasis);
```

Comment:
Ah wait, now I do get an error in browser console about this:
```
main.js?v=0.43.1b1:5104 Uncaught ReferenceError: actionButtonWrapper is not defined
    at addCameraFrameUi (main.js?v=0.43.1b1:5104:13)
    at updateCameras (main.js?v=0.43.1b1:5133:13)
    at recreateCameraFrames (main.js?v=0.43.1b1:5148:9)
    at HTMLImageElement.<anonymous> (main.js?v=0.43.1b1:3396:21)
    at r.complete (jquery.min.js?v=0.43.1b1:2:62910)
    at c (jquery.min.js?v=0.43.1b1:2:28327)
    at Object.fireWith [as resolveWith] (jquery.min.js?v=0.43.1b1:2:29072)
    at u (jquery.min.js?v=0.43.1b1:2:59642)
    at S.fx.tick (jquery.min.js?v=0.43.1b1:2:64608)
    at ot (jquery.min.js?v=0.43.1b1:2:58967)
```
And here the asset versioning is visible, so all good in this regards 🙂.

Reply:
The reason why I need a wrapper for every single button is that I wanted to space all buttons evenly. So each element needs to be able to grow. Without this, it would look like this:
![192 168 1 218_8765_](https://github.com/motioneye-project/motioneye/assets/32551072/59fe733f-66dd-4833-99d5-5c0d82aa08a4)
And I can't do this on the underlying div that contains the action button itself because they contain a background-image of the buttons. So when you change the dimension of this div, it would look something like this:
![192 168 1 218_8765_ (1)](https://github.com/motioneye-project/motioneye/assets/32551072/faab15c9-b4e1-4fc3-b87b-255ba31b1636)

Reply:
Understood. So in case we want to reduce the amount of nested elements, we'd need to do more changes to how the buttons are presented, like splitting the images at best into individual svg paths, directly inline attached to an individual class each. Much cleaner anyway than this strange background positioning. But something we can address in a dedicated PR when someone finds time and mood for some non-visual optimisations.

But more importantly, what about the missing `actionButtonWrapper`? That one still needs to be defined somewhere, or CSS selector used directly like in my suggestion.

Reply:
Oh, don't know why but it got deleted for some reason. I put it back in. 

So I played with CSS again a bit but I see no way that one can do it without a wrapper as long as all this icons are in one image. It would be easy if the buttons were a fixed size all the time but because the width of the wrapper changes depending on the button count, it is not possible like this. Right now every button has it's own class already. I think it wouldn't be too big of a trouble to fix it. I can do it in the next PR if you want.

Reply:
So I ended up splitting the button SVG after all, so we have a clean solution now instead of one more thing to think about. So I could also remove the `actionButtonWrapper `. All in all a lot cleaner.